src/moda_bitbuf.F

```diff
@@ -9,20 +10,16 @@
 
```

Comment:
Just for consistency, maybe author+date should go in the same line?

Reply:
I noticed that as well, and tested it. Turns out that doxygen does the same thing whether those two tags are on the same line as not.

Reply:
Did you mean to leave in the original author and date?  It looks like you have that information repeated now in the file, on line 3 (where you've now combined them onto one line) and on lines 8 and 9 (where they originally were, and still on separate lines).  How do you know which one is taking effect in the HTML output?

Reply:
Never mind, going back now and looking at the [HTML output from the `develop` branch](https://emc.ncep.noaa.gov/emc/pages/infrastructure/bufrlib_jeff/html_new/moda__bitbuf_8F.html), it looks like the author and date weren't showing up there at all, but I see they are now showing up in the [HTML output from your `ejh_docs_1` branch](https://emc.ncep.noaa.gov/emc/pages/infrastructure/bufrlib_jeff/html_PR253/moda__bitbuf_8F.html).  So obviously this must have triggered from your new line 3.

So all good, but can you please remove the duplication (i.e. my original version of that information) from lines 8 and 9?

Reply:
In case you missed my earlier comment, and if we're going to now list the author and date on line 3 in the file block, can we please remove the duplicate author and date tags from lines 8 and 9?

Reply:
I think the idea is to have separate author/date for the file and for the module. I don't love the redundancy but I think this is correct.

Reply:
Even when the module is the only thing in the file?  Or is this related to the changes being proposed in #252?

Reply:
That's my understanding, yes. If we start putting in multiple subroutines etc. per file (#252), then I think this will make a little more sense :)